builds/e2e/templates/deploy-iotuap-setup.yaml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  release.label: ''
+  edgelet.artifact.name: ''
+  images.artifact.name: ''
+  container.registry: ''
+  container.registry.username: ''
+  container.registry.password: ''
+  iotHub.connectionString: ''
+  eventHub.connectionString: ''
+  snitch.build.number: ''
+  snitch.alert.url: ''
+  snitch.storage.account: ''
+  snitch.storage.masterKey: ''
+  amqp.settings.enabled: 'true'
+  mqtt.settings.enabled: 'true'
+  loadGen.message.frequency: ''
+  loadGen1.transportType: 'Amqp_Tcp_Only'
+  loadGen2.transportType: 'Amqp_WebSocket_Only'
+  loadGen3.transportType: 'Mqtt_Tcp_Only'
+  loadGen4.transportType: 'Mqtt_WebSocket_Only'
+
+steps:
+  - task: CopyFiles@2
+    displayName: 'Copy Edgelet Artifact'
+    inputs:
+      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['edgelet.artifact.name'] }}"
+      TargetFolder: "\\\\$(iotuap-x64-machine)\\c$\\Data\\e2etests\\artifacts\\${{ parameters['edgelet.artifact.name'] }}"
+      CleanTargetFolder: true
+  - task: CopyFiles@2
+    displayName: 'Copy Images Artifact'
+    inputs:
+      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['images.artifact.name'] }}"
+      TargetFolder: "\\\\$(iotuap-x64-machine)\\c$\\Data\\e2etests\\artifacts\\${{ parameters['images.artifact.name'] }}"
+      CleanTargetFolder: true
+  - task: PowerShellOnTargetMachines@3
+    displayName: 'Update iotedged'
+    inputs:
+      Machines: '$(iotuap-x64-machine)'
+      UserName: '$(iotuap-x64-login)'
+      UserPassword: '$(iotuap-x64-password)'
+      InlineScript: |
```

Comment:
please wrap this inline script in existing E2E test script.

Reply:
I do not think we have an existing E2E test script. Everything for E2E is done as inline PS commands in the stages.
We can do that (wrap all inline scripts in one test script) as a separate pull request, to not over-complicate this one.

Reply:
Please check existing Windows x64; each step mainly call to RunE2ETests.ps1 only.  Therefore this inline script should move  into this script.

Reply:
I moved the inline script to RunE2ETests.ps1, feedback is welcome